project.py

```diff
@@ -84,52 +119,16 @@
     # вік
     Age = int(data2[0])
 
-    if Age < AGE_20:
-        Sympthoms2[1, 1] += POINT_1
-    elif (Age >= AGE_20) and (Age < AGE_40):
-        Sympthoms2[1, 2] += POINT_1
-    elif (Age >= AGE_40) and (Age < AGE_60):
-        Sympthoms2[1, 3] += POINT_1
-    else:
-        Sympthoms2[1, 4] += POINT_1
-
-    # блювота
-    if data2[1] == 'yes':
-        Sympthoms2[2, 1] += POINT_1
-    else:
-        Sympthoms2[2, 2] += POINT_1
-
-    # жовтушність
-    if data2[2] == 'eye':
-        Sympthoms2[3, 1] += POINT_1
-    elif data2[2] == 'skin':
-        Sympthoms2[3, 2] += POINT_1
-    else:
-        Sympthoms2[3, 3] += POINT_1
-
-    # напади болю
-    if data2[3] == 'yes':
-        Sympthoms2[4, 1] += POINT_1
-    else:
-        Sympthoms2[4, 2] += POINT_1
-
-    # збільшення печінка
-    if data2[4] == 'yes':
-        Sympthoms2[5, 1] += POINT_1
-    else:
-        Sympthoms2[5, 2] += POINT_1
-
-    # апетит
-    if data2[5] == 'yes\n':
-        Sympthoms2[6, 1] += POINT_1
-    else:
-        Sympthoms2[6, 2] += POINT_1
+    Sympthoms2 = check_age(Age, Sympthoms2)
+    Sympthoms2 = check_vomit(data2, Sympthoms2)
+    Sympthoms2 = check_yellowness(data2, Sympthoms2)
+    Sympthoms2 = check_pain(data2, Sympthoms2)
+    Sympthoms2 = check_liver(data2, Sympthoms2)
+    Sympthoms2 = check_appetite(data2, Sympthoms2)
 
 PrK2 = Sympthoms2 / Diagnose2
 print(PrK2)
 
-
-
 Sympthoms3 = np.zeros((7, 5))
 text_file = open("Hepatitis_var5.csv", "r")
 lin3 = text_file.readlines()
```

Comment:
Is it possible to put the program logic after opening the file into a function and use it after opening each file? For example, as in the lines: 84-105, 107-130,133-155